src/lang/lang_lexer.ml

```diff
@@ -162,6 +162,9 @@ let rec token lexbuf =
     | "ref" -> REF
     | "def" -> PP_DEF
     | "replaces" -> REPLACES
+    | "try" -> TRY
+    | "catch" -> CATCH
+    | "do" -> DO
```

Comment:
Aren't there some other files to edit when adding new syntax? I saw some in your other PR, I'm just not familiar with this.

Reply:
liquidsoap.xml in the documentation (syntax coloration) and the emacs mode.